clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/TransferWgsCramV1Metadata.scala

```diff
@@ -59,12 +59,13 @@ case class TransferWgsCramV1Metadata(
   readgroupLevelMetricsFiles: Option[List[URI]] = None
 ) extends TransferMetadata[TransferWgsCramV1Metadata] {
 
-  // As of DSDEGP-1711, we are only delivering the cram, crai, and md5
+  // As of DSDEGP-1711, delivering only the cram and crai.
```

Comment:
My understanding is that the MD5 is still delivered but it isn't done here. Maybe having a comment about that here isn't very helpful.

Reply:
It could be reworded to say "As of DSDEGP-1711, no longer delivering metrics"